src/brain/issueLabelHandler/index.test.ts

```diff
@@ -748,5 +768,37 @@ describe('issueLabelHandler', function () {
         org.project.fieldIds.responseDue
       );
     });
+
+    it.each([
+      WAITING_FOR_PRODUCT_OWNER_LABEL,
+      WAITING_FOR_SUPPORT_LABEL,
+      WAITING_FOR_COMMUNITY_LABEL,
+    ])(
+      "should clear project issue status field if user removes '%s'",
+      async function (label) {
+        await setupIssue();
+        await addLabel(label, 'routing-repo');
+        await removeLabel(label);
```

Comment:
nit: we only use this in two places, here and the test below. But the actual object its building to put in `makePayload` is actually pretty dramatically different - enough that you have 3 args for a function that only gets used twice. Imo its easier to read, and makes it clear what the test is doing.

Classic memo on not trying to make tests to DRY seems pretty applicable here: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
fair enough, inlined